scipy/stats/tests/test_distributions.py

```diff
@@ -3258,6 +3258,26 @@ def test_ncx2_tails_pdf():
     assert_(np.isneginf(logval).all())
 
 
+@pytest.mark.parametrize('method, expected', [
+    ('cdf', np.array([1.72115630e-04, 2.74102033e-05])),
+    ('pdf', np.array([0.00078975, 0.00013013])),
+    ('logpdf', np.array([-7.14378973, -8.94700192]))
+])
+def test_ncx2_zero_nc(method, expected):
+    # gh-5441
+    # ncx2 with nc=0 is identical to chi2
+    result = getattr(stats.ncx2, method)(1, nc=[0, 4], df=10)
```

Comment:
Can just parameterize with ncx2.pdf, ncx2.cdf, ncx2.logpdf and get rid of the getattr?

Reply:
The parameter of the pytest decorator is regarded as `string` which doesn't have `__call__`.
Explicitly, if the parameters is set as follows:
```
@pytest.mark.parametrize('method, expected', [
    ('ncx2.cdf', np.array([2.497951336e-09, 3.437288941e-10])),
    ('ncx2.pdf', np.array([1.238579980e-07, 1.710041145e-08])),
    ('ncx2.logpdf', np.array([-15.90413011, -17.88416331]))
])
```
`method(0.1, nc=[0, 4], df=10)` would run into `TypeError`.